src/app/store.ts

```diff
@@ -1,6 +1,7 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import `combineSlices` should be `combineReducers` from `@reduxjs/toolkit`. `combineSlices` is not a valid function in Redux Toolkit.